src/pages/profile/external-identities/FenceAccount.test.ts

```diff
@@ -0,0 +1,160 @@
+import { act, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { h } from 'react-hyperscript-helpers';
+import { Ajax } from 'src/libs/ajax';
+import { authStore, FenceStatus } from 'src/libs/state';
+import { FenceAccount } from 'src/pages/profile/external-identities/FenceAccount';
+import { asMockedFn, renderWithAppContexts } from 'src/testing/test-utils';
+
+// Mocking for Nav.getLink
+jest.mock('src/libs/nav', () => ({
+  ...jest.requireActual('src/libs/nav'),
```

Comment:
For added type safety in mocks, consider something like this:

```suggestion
type NavExports = typeof import('src/libs/nav');
jest.mock('src/libs/nav', (): NavExports => ({
  ...jest.requireActual<NavExports>('src/libs/nav'),
```